src/server/replica.cc

```diff
@@ -1077,6 +1080,9 @@ error_code Replica::ParseReplicationHeader(base::IoBuf* io_buf, PSyncResponse* d
 auto Replica::GetSummary() const -> Summary {
   auto f = [this]() {
     auto last_io_time = LastIoTime();
+
+    // Note: we access LastIoTime from foreigh thread in unsafe manner. However, specifically here
```

Comment:
Then why don't we just make LastIoTime atomic ?

Reply:
I think in general the mix of share nothing, message passing and atomic is very confusing.
In any case, it's out of scope of this PR - I just surfaced a potential problem here.

Reply:
+1 for out of scope can be fixed on separate PR. IMO nothing really confusing here. It's a data race between reading flow->LastIoTime() from one thread and updating this from another.